src/AutoRest.CSharp/Common/Input/InputTypes/InputType.cs

```diff
@@ -36,6 +37,9 @@ internal InputType GetCollectionEquivalent(InputType inputType)
 
     //public bool IsNullable { get; init; }
     public string Name { get; internal set; }
+
+    public IReadOnlyList<InputDecoratorInfo> Decorators { get; internal set; } = new List<InputDecoratorInfo>();
```

Comment:
`Decorators` is required property, shall we add this in constructor?

Reply:
It is not needed in constructor since it is default to an empty list.
We only need deserialization to parse it from TCGC output.